src/test/resources/application.yml

```diff
@@ -0,0 +1,18 @@
+spring:
+  jpa:
+    database: H2
+    show-sql: true
```

Comment:
to be moved to a application-default.yml file with only the overriden keys like in the recent study-server pr

Reply:
Done